esp/bindings/http/platform/httptransport.cpp

```diff
@@ -773,6 +773,8 @@ void CHttpMessage::logMessage(MessageLogFlag messageLogFlag, const char *prefix)
                 DBGLOG("%s<non-text content or content type not specified>", prefix);
             else if ((m_content_type.length() > 0) && (strieq(m_content_type.get(), "text/css") || strieq(m_content_type.get(), "text/javascript")))
                 DBGLOG("%s<content_type: %s>", prefix, m_content_type.get());
+            else if (m_content.length() < 80)
```

Comment:
This may not be a good choice. The code here tries to hide the password only if the content is from our ECLWatch login page (the content of the login page is < 80 bytes). I create another JIRA to fix this: https://track.hpccsystems.com/browse/HPCC-17809.